crates/common/consensus/src/electra/beacon_state.rs

```diff
@@ -1593,6 +1627,80 @@ impl BeaconState {
         Ok(())
     }
 
+    pub fn compute_sync_committee_rewards(
+        &self,
+        beacon_block: &SignedBeaconBlock,
+    ) -> anyhow::Result<HashMap<u64, u64>> {
+        let sync_aggregate = &beacon_block.message.body.sync_aggregate;
+        let sync_committee = &self.current_sync_committee;
+
+        let sync_committee_indices = self.get_sync_committee_indices(sync_committee)?;
+
+        let (participant_reward, proposer_reward) = self.get_proposer_and_participant_rewards();
+
+        let mut balances = HashMap::<usize, u64>::new();
+
+        for validator_index in &sync_committee_indices {
+            balances.insert(
+                *validator_index,
+                *self
+                    .balances
+                    .get(*validator_index)
+                    .ok_or(anyhow!(String::from("Sync Committee Rewards Sync Error",)))?,
```

Comment:
```suggestion
                    .ok_or(anyhow!("Sync Committee Rewards Sync Error"))?,
```

why is `String::from` here? same for other places in the code

Reply:
checked that we don't require it, removed.